examples/ios/Examples/BundleRealms.swift

```diff
@@ -0,0 +1,118 @@
+// :replace-start: {
+//   "terms": {
+//     "QsTask": "Task"
+//   }
+// }
+
+import XCTest
+import RealmSwift
+import Foundation
+
+class BundleRealms: XCTestCase {
+
+    func testCopyRealm() async throws {
+        // :code-block-start: copy-synced-realm-for-bundling
+        let app = App(id: YOUR_REALM_APP_ID)
+
+        // Log in the user whose Realm you want to copy for bundling
```

Comment:
```suggestion
        // Log in the user whose realm you want to copy for bundling
```
maybe i should just give up the lowercase realm effort

Reply:
Hah, no, you're right, I'm really bad about consistency w/Realm vs. realm. I'll clean up the instances in this PR. I should really make a checklist for myself to run through when making PRs and add this to it...